src/frontend/js/components/ReferralTable/index.tsx

```diff
@@ -50,7 +50,7 @@ const messages = defineMessages({
     id: 'components.ReferralTable.object',
   },
   requesters: {
-    defaultMessage: 'Requesters',
+    defaultMessage: "Unit's Requesters",
```

Comment:
```suggestion
    defaultMessage: "Requesters' unit(s)",
```

Reply:
it could be "Requester's unit(s)" ?

Reply:
`"Requesters' unit(s)"` means several requesters, `"Requester's unit(s)"` just one requester